tests/cmdline/commands/test_computer.py

```diff
@@ -511,6 +515,72 @@ def test_show(self):
         assert '--username=' in result.output
         assert result_cur.output == result.output
 
+    @pytest.mark.parametrize('sort', ['--sort', '--no-sort'])
+    def test_computer_export_setup(self, tmp_path, sort):
+        """Test if 'verdi computer export setup' command works"""
+        self.comp_builder.label = 'test_computer_export_setup' + sort
+        self.comp_builder.transport = 'core.ssh'
+        comp = self.comp_builder.new()
+        comp.store()
+
+        exported_setup_filename = tmp_path / 'computer-setup.yml'
+        result = self.cli_runner(computer_export_setup, [sort, comp.label, exported_setup_filename])
+        assert result.exit_code == 0, 'Command should have run successfull.'
+        assert f'{exported_setup_filename}' in result.output, 'Filename should be in terminal output but was not found.'
+        assert os.path.exists(exported_setup_filename), f"'{exported_setup_filename}' was not created during export."
```

Comment:
Nitpick, but `exported_setup_filename` is a `pathlib.Path` instance so we should prefer that built in API:
```suggestion
        assert exported_setup_filename.exists(), f"'{exported_setup_filename}' was not created during export."
```
This table mapping `os` operations to `pathlib` is handy: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module